app/javascript/packages/document-capture/components/acuant-capture.tsx

```diff
@@ -399,7 +400,7 @@ function AcuantCapture(
         size: nextValue.size,
         failedImageResubmission: hasFailed,
       });
-      trackEvent(`IdV: ${name} image added`, analyticsPayload);
+      trackEvent(`IdV: ${name} image added`, { ...analyticsPayload, acuantCaptureMode: null });
```

Comment:
Can we incorporate this into the logic of `getAddAttemptAnalyticsPayload`, especially since we already pass `source` to that function?

Something like...

```ts
  /**
   * Returns an analytics payload, decorated with common values.
   */
  function getAddAttemptAnalyticsPayload<
    P extends ImageAnalyticsPayload | AcuantImageAnalyticsPayload,
  >(payload: P): P {
    const enhancedPayload = { ...payload, attempt };
    if (payload.source === 'acuant') {
      enhancedPayload.acuantCaptureMode = acuantCaptureMode;
    }
    incrementAttempt();
    return enhancedPayload;
  }
```